NSelene/SeleneElement.cs

```diff
@@ -366,5 +382,30 @@ public static SeleneCollection SS(this SeleneElement selement, string cssSelecto
                 return selement.FindAll(cssSelector);
             }
         }
+
+        public static class JSElementExtensions
```

Comment:
Let's add these extensions to SeleneElementExtensions for simplicity... to keep all element extensions in one place...

Reply:
Fine.